magento/adobe-stock-integration
# #575 : Licensed overlay labels are not updated when licensing and login/logout

### Description (*)

This PR will introduce the feature to display license label after login and log out from Adobe IMS
### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#575: Licensed overlay labels are not updated when licensing and login/logout

### Manual testing scenarios (*)

1. Login to admin panel
2. Open Magento Media Gallery (i.e. go to Cms -> Pages, edit the page and insert image)
3. Click "Search Adobe Stock" button to open images grid
4. Log in to Adobe Stock (previously licensed images should be marked as "Licensed" in grid)
5. License an image (just licensed image should be marked as licensed in grid)
6. Sign out from Adobe Stock
7. Licensed labels will be removed from grid
